ddl/ddl_worker.go

```diff
@@ -393,6 +393,14 @@ func (w *worker) handleDDLJobQueue(d *ddlCtx) error {
 				err = w.finishDDLJob(t, job)
 				return errors.Trace(err)
 			}
+			if runJobErr != nil && !job.IsRollingback() && !job.IsRollbackDone() {
```

Comment:
This line of code is different from that on the master branch. Maybe we can wait for the new PR in the master branch

Reply:
great, done.